src/pages/OfferNew/Checkout/Sign.tsx

```diff
@@ -137,12 +136,12 @@ export const Sign: React.FC<Props> = ({
             <MarkdownTranslation
               textKey="CHECKOUT_SIGN_DISCLAIMER"
               replacements={{
-                PREBUY_LINK: textKeys[
-                  getPrebuyPDFTextKey(quote.contractType)
-                ](),
-                TERMS_LINK: textKeys[
-                  getInsurancePDFTextKey(quote.contractType)
-                ](),
+                PREBUY_LINK: quote.insuranceTerms.get(
+                  InsuranceTermType.PreSaleInfo,
+                )!!.url,
+                TERMS_LINK: quote.insuranceTerms.get(
+                  InsuranceTermType.TermsAndConditions,
+                )!!.url,
```

Comment:
This will have to be updated to satisfy the Norway specific text

Reply:
`isSwedish` then one case and `isNorwegian` another

Reply:
Oh no, it doesn't work or?